File: java/src/model/ataque/Zapatazo.java
package model.ataque;

import model.chimpos.Chinpokomon;

import java.util.Random;

public class Zapatazo extends Ataque {

    public Zapatazo() {
        super(1, "Zapatazo", 3);
    }

    public void atacar(Chinpokomon chipo, Integer valor) {
        Random random = new Random();
        int value = random.nextInt(2 + 1) + 1;
        if (value == 1) {
            chipo.recibirDanio(valor * 2);
        } else {
            chipo.recibirDanio(valor);
        }
    }

}

File: python/model/ataque/Zapatazo.py
from random import randint
from model.ataque.Ataque import Ataque

class Zapatazo (Ataque):
        
        def __init__(self):
            Ataque.__init__(self, 1, "Zapatazo", 3)
        
        def atacar_con_valor(self, chipo, valor):
            random = randint(1, 2)
            if(random == 1):
                chipo.recibir_danio(valor * 2)
            else:
                chipo.recibir_danio(valor)